AppDashboard/lib/app_dashboard_data.py

```diff
@@ -647,3 +721,45 @@ def can_upload_apps(self):
     except Exception as err:
       logging.exception(err)
       return False
+
+  def get_dash_layout_settings(self, email=None):
+    """ Queries the UserAppServer to see what settings the user has saved
+    for customizing the UI of the Dashboard.
+
+    Args:
+      email: A str that indicates the e-mail address of the administrator of
+        this application, or None if the currently logged-in user is the admin.
+
+    Returns:
+      A JSON string describing the customization layout
+    """
+    LOOKUP_DICT = self.build_dict()
+    #base admin template
+    default_dash_layout ='''{
+        "nav":["app_management","appscale_management","debugging_monitoring"],
+        "panel":["cloud_stats","database_stats","memcache_stats"]
+        }'''
+    temp_dict = json.loads(default_dash_layout)
+    temp_dict['nav'][:] = [{key:LOOKUP_DICT.get(str(key))[1]} for key in
```

Comment:
The slice syntax is only necessary if you need to modify the list itself (eg. if some other variable is referencing it directly and you need to change that reference). Since you are creating a whole new list for this key here, you can just set it directly with `temp_dict['nav'] = ...`.

All of the keys are strings anyway, so there's no need for the `str` function.